src/forcing/ForcingsEngineLumpedDataProvider.cpp

```diff
@@ -118,19 +148,39 @@ Provider::data_type Provider::get_value(
         auto s_t  = std::chrono::system_clock::to_time_t(start);
         auto e_t  = std::chrono::system_clock::to_time_t(end);
 
-        std::cout << "get_value() Time begin: " << std::put_time(std::gmtime(&tb_t), "%Y-%m-%d %H:%M:%S UTC")
-                  << " (" << tb_t << ")"
-                  << " | start: " << std::put_time(std::gmtime(&s_t), "%Y-%m-%d %H:%M:%S UTC")
-                  << " (" << s_t << ")" << std::endl;
+        ss.str(""); 
+        ss << "get_value() Time begin: " << std::put_time(std::gmtime(&tb_t), "%Y-%m-%d %H:%M:%S UTC")
+            << " (" << tb_t << ")"
+            << " | start: " << std::put_time(std::gmtime(&s_t), "%Y-%m-%d %H:%M:%S UTC")
+            << " (" << s_t << ")" << std::endl;
+        LOG(LogLevel::INFO, ss.str());
```

Comment:
```suggestion
        LOG(LogLevel::DEBUG, ss.str());
```
This message and the one immediately below should probably be `DEBUG`, because this function is going to get called every timestep, for every catchment, once per requested variable.

We maybe don't even need logging here at all, but I won't push for that.

Reply:
Updated.